### PR TITLE
Rename create-example to create-workleap add scripts and some more cleanup ;; WLFE-22

### PR DESCRIPTION
**Pull request by ofrogon**
Also:
- Implement new Write method in prompts
- Remove some "template" code.

At the same time, in this PR I'm validating the project branch policies.

### CODE REVIEW

**Review by alexasselin008 — APPROVED** [review]

**Comment by patricklafrance** [comment]
From my experience, while doing this type of coding, it's usually easier to use "scripting" coding style, meaning just write the code in a few files and a few functions as if it's a script that read from top to bottom.

Trying to introduce patterns and Classes tend to makes it harder and less flexible.

To prove my point, have a look at how those project are doing it:

- [create-svelte](https://github.com/sveltejs/kit/blob/master/packages/create-svelte/bin.js)
- [create-vue](https://github.com/vuejs/create-vue/blob/main/index.ts)
- [create-vite](https://github.com/vitejs/create-vite-app/blob/master/index.js)
- [create-next-app](https://github.com/vercel/next.js/blob/canary/packages/create-next-app/index.ts)

File: packages/create-workleap/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@foundry-cli/create-example",
+  "name": "@foundry-cli/create-workleap",
```
[the remark this answers is not in the document]
Comment (franckgaudin) [reply packages/create-workleap/package.json:2]: interesting, that's good to know, thanks @alexasselin008

[the remark this answers is not in the document]
Comment (ofrogon) [reply packages/create-workleap/package.json]: Yea, the way we want to name this package was decided this P.M. and this commit wasn't reflecting that change at that moment.  Last commit reflect that choice.

**Comment by ofrogon** [comment]
@patricklafrance I will apply the changes to a more "scripting" style in a future PR, I want to add the tests I worked on last week first... and they are waiting in a branch that requires the changes I made here.

File: README.md [hunk 2]
```diff
@@ -2,12 +2,37 @@
...
-Put it on the parking lot this proposal is a win-win situation which will cause a stellar paradigm shift, and produce a multi-fold increase in deliverables. Finance on-brand but completeley fresh, or who's responsible for the ask for this request?, but let's put a pin in that, so critical mass. In an ideal world locked and loaded prairie dogging workflow ecosystem. Open door policy no scraps hit the floor. This is a no-brainer. Reach out face time.
\ No newline at end of file
+## Usage
+
+To create a new project base on the Workleap starter kit, run:
+
+### npm
+
+```cmd
+npm create @foundry-cli/workleap <your-app-name>
```
Comment (franckgaudin) [inline README.md:21]: are the orders still valid with the package `create-workleap` ?

**Review by alexasselin008 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ofrogon) [reply README.md:21]: Ha no... I forgot to update that file.

**Review by alexasselin008 — APPROVED** [review]

**Pull request merged**